reana_server/rest/workflows.py

```diff
@@ -1113,9 +1126,14 @@ def get_workflow_status(workflow_id_or_name, user):  # noqa
     try:
         if not workflow_id_or_name:
             raise ValueError("workflow_id_or_name is not supplied")
+        include_last_command = json.loads(
+            request.args.get("include_last_command", "false").lower()
+        )
```

Comment:
What about validating and parsing parameters with `use_args`, as done in the other endpoint?